pkg/networkutils/network.go

```diff
@@ -273,39 +270,7 @@ func (n *linuxNetwork) SetupHostNetwork(vpcv4CIDRs []string, primaryMAC string,
 	v4Enabled bool, v6Enabled bool) error {
 	log.Info("Setting up host network... ")
 
-	var err error
 	primaryIntf := "eth0"
-	//RP Filter setting is only needed if IPv4 mode is enabled.
-	if v4Enabled && (n.nodePortSupportEnabled || !n.useExternalSNAT) {
-		primaryIntf, err = findPrimaryInterfaceName(primaryMAC)
-		if err != nil {
-			return errors.Wrapf(err, "failed to SetupHostNetwork")
-		}
-		// If node port support is enabled, configure the kernel's reverse path filter check on eth0 for "loose"
-		// filtering. This is required because
-		// - NodePorts are exposed on eth0
-		// - The kernel's RPF check happens after incoming packets to NodePorts are DNATted to the pod IP.
-		// - For pods assigned to secondary ENIs, the routing table includes source-based routing. When the kernel does
-		//   the RPF check, it looks up the route using the pod IP as the source.
-		// - Thus, it finds the source-based route that leaves via the secondary ENI.
-		// - In "strict" mode, the RPF check fails because the return path uses a different interface to the incoming
-		//   packet. In "loose" mode, the check passes because some route was found.
-		//
-		// The same applies when performing SNAT on Pod traffic.
-		primaryIntfRPFilter := "net/ipv4/conf/" + primaryIntf + "/rp_filter"
-		const rpFilterLoose = "2"
-
-		if n.shouldConfigureRpFilter {
```

Comment:
Also cleanup shoudConfigureRpFilter attribute, the underlying env variable and documentation.